examples/census-hv-dask.ipynb

```diff
@@ -170,6 +184,7 @@
    },
    "outputs": [],
    "source": [
+    "shaded = datashade(census_points, **shade_defaults)\n",
```

Comment:
Duplicated here to avoid inappropriate sharing of streams? Ok.

Reply:
Yes.